.circleci/config.yml

```diff
@@ -8,27 +8,26 @@ jobs:
           name: Install Dependencies
           command: |
             apt-get update
-            apt-get install cabal-install
+            apt-get install -y cabal-install
 
       - checkout
 
-      - run:
-          name: workaround for certificate failure
-          command: |
-            mv /etc/apt/sources.list.d/nodesource.list /etc/apt/sources.list.d/nodesource.list.disabled
-            apt-get update
-            apt-get -y upgrade
-            apt-get  install -y ca-certificates libgnutls30
-            mv /etc/apt/sources.list.d/nodesource.list.disabled /etc/apt/sources.list.d/nodesource.list
-
       - run:
           name: Install pip
           command: apt-get install -y python3-pip
 
       - run:
           name: Install dependencies
-          command: pip install --user -r requirements.txt
+          command: pip3 install --user -r requirements.txt
+
+      - run:
+          name: upgrade Cabal
+          command: |
+            cabal update
+            cabal install cabal-install
```

Comment:
is there a particular version we think we need (perhaps a version newer than what's in the Debian package db)?  Can we request that specifically?  Or even if we just want some newer version, it would be nice to ask for one specifically, if we can, to improve reproducibility.